ActivityPub.Types/AS/Collection/ASCollection.cs

```diff
@@ -98,6 +99,32 @@ public int TotalItems
     public bool HasItems => Items?.Any() == true;
 
     public static implicit operator ASCollection(List<ASObject> collection) => new() { Items = new LinkableList<ASObject>(collection) };
+
+    public IEnumerator<Linkable<ASObject>> GetEnumerator()
```

Comment:
Can this be simplified to pass through the iterator from `Items`? Something like this:
```suggestion
    public IEnumerator<Linkable<ASObject>> GetEnumerator()
        => Items?.GetEnumerator() ?? Enumerable.Empty<Linkable<ASObject>>();
```

Reply:
Yeah this should totally be doable! Great idea, thanks! Can't believe I didn't think of that